### DISCUSSION TITLE
<deviceID> ?

### DISCUSSION THREAD

**Discussion by hape62** [z-master42/solarflow#41] [General]
Ich stehe auf dem Schlauch und frag daher mal nach, womit ersetze ich die deviceID in der mqtt.yaml?

**Comment by z-master42** [comment]
Ich gehe mal davon aus, dass du die Anleitung gelesen hast. Dann hast du es da wohl überlesen. Steht unter 2. i. In dem Teil mit dem MQTT-Explorer.

**Reply by hape62** [reply]
Das hatte ich tatsächlich überlesen mir aber schon gedacht, dass es dieser Wert sein muss. 
Hatte ihn daher testweise auch schon mal eingefügt, aber ich bekam immer noch keine Werte im HA angezeigt.
Aber jetzt bin ich schon mal ein Stück weiter. Danke.
